src/test/java/bio/terra/service/duos/DuosServiceTest.java

```diff
@@ -64,9 +157,7 @@ public void testCreateFirecloudGroupWithNamingConflict() {
     // we don't know exactly what our backstop name will be as it will be suffixed by a new UUID,
     // but we know that it will start with our more readable group name.
     when(iamService.createGroup(startsWith(FIRECLOUD_GROUP_NAME))).thenReturn(groupEmailNew);
-    IamConflictException iamConflictEx =
-        new IamConflictException("Group already existed", List.of());
-    doThrow(iamConflictEx).when(iamService).createGroup(FIRECLOUD_GROUP_NAME);
+    doThrow(mock(IamConflictException.class)).when(iamService).createGroup(FIRECLOUD_GROUP_NAME);
```

Comment:
I'm just curious why change to a mock from an instance?

Reply:
Whoops, missed this question.

I was interested in minimizing definitions and object creation for tests where the internals aren't useful.  In this case, we don't need to check or do anything with the contents of the exception; its class is what dictates the behavior we're testing.

Reply:
Personally I think I wouldn't mock a non-service class like an exception, unless you needed to mock a method on it. I would write something like
```suggestion
    doThrow(new IamConflictException("", List.of())).when(iamService).createGroup(FIRECLOUD_GROUP_NAME);
```
so it's clear that the contents aren't used

Reply:
That's a good point, I'll do that instead!